src/_premake_init.lua

```diff
@@ -1512,6 +1512,39 @@
 		tokens = true
 	}
 
+	api.register {
```

Comment:
Given these are VS only APIs, they should be registered by the VS module, rather than the global API registry.

Reply:
Good call, I definitely didn't put the feature in the global table without even noticing that a module-specific one exists :wink: